match/lib/match/runner.rb

```diff
@@ -221,18 +196,19 @@ def fetch_provisioning_profile(params: nil, certificate_id: nil, app_identifier:
       end
 
       profile_name = names.join("_").gsub("*", '\*') # this is important, as it shouldn't be a wildcard
-      base_dir = File.join(prefixed_working_directory(working_directory), "profiles", prov_type.to_s)
+      base_dir = File.join(prefixed_working_directory, "profiles", prov_type.to_s)
       profiles = Dir[File.join(base_dir, "#{profile_name}.mobileprovision")]
       if Helper.mac?
         keychain_path = FastlaneCore::Helper.keychain_path(params[:keychain_name]) unless params[:keychain_name].nil?
       end
 
       # Install the provisioning profiles
       profile = profiles.last
+      force = params[:force]
 
       if params[:force_for_new_devices] && !params[:readonly]
         if prov_type != :appstore
-          params[:force] = device_count_different?(profile: profile, keychain_path: keychain_path, platform: params[:platform].to_sym) unless params[:force]
+          force = device_count_different?(profile: profile, keychain_path: keychain_path, platform: params[:platform].to_sym) unless params[:force]
```

Comment:
Long line + a trailing negative conditional makes it hard to understand. Lets set `force` in an `if/else` block.

Reply:
Oh god, I didn't even see that railing negative conditional, TBH... refactor happening 💪